### PR TITLE
Get entire Cex.IO order book, handle sequence number mismatch smoothly

### PR DESCRIPTION
**Pull request by BenjaminTrapani**
Fixes an issue where the market data reader would crash due to trying to resubscribe to too many channels. Also makes entire order book available by default.

### CODE REVIEW

File: xchange-cexio/src/main/java/info/bitrich/xchangestream/cexio/CexioStreamingMarketDataService.java [hunk 1]
```diff
@@ -39,17 +39,15 @@ public OrderBookUpdateConsumer(CexioStreamingRawService streamingOrderDataServic
...
-      if (prevID == null || prevID.add(BigInteger.ONE).compareTo(t.id) == 0) {
-        prevID = t.id;
-        orderBookSoFar = CexioAdapters.adaptOrderBookIncremental(orderBookSoFar, t);
-        retVal = orderBookSoFar;
-      } else {
-        streamingOrderDataService.resubscribeChannels();
-        retVal = orderBookSoFar;
-        orderBookSoFar =
+      if (prevID != null && prevID.add(BigInteger.ONE).compareTo(t.id) != 0) {
+        OrderBook orderBookSoFar =
```
Comment (badgerwithagun) [inline xchange-cexio/src/main/java/info/bitrich/xchangestream/cexio/CexioStreamingMarketDataService.java]: This line doesn't look like it does anything! Just assigns a local variable which is never used.

Comment (BenjaminTrapani) [reply xchange-cexio/src/main/java/info/bitrich/xchangestream/cexio/CexioStreamingMarketDataService.java]: Lol good catch, that's what I get for copy-pasting the assignment in the declaration. The statement is supposed to assign to the member variable, fixed.

**Comment by badgerwithagun** [comment]
@dozd, the webhook from Travis doesn't seem to have fired and GitHub won't let me merge this. Would you mind?

**Comment by badgerwithagun** [comment]
Please note that I am unable to merge PRs at present due to an issue with Travis not notifying GitHub that the build ran successfully. The repository is set to disallow merges if checks are in progress/failing so merging is disabled.

This needs @dozd's attention.

**Pull request merged**